.github/README-pt-BR.md

```diff
@@ -47,9 +47,10 @@ NES.css não fornece nenhuma fonte, mas nós mantemos uma lista de fontes recome
 | -------- | ------------------------------------------------------------------ |
 | (Padrão) | [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P) |
 | Inglês   | [Kongtext](https://www.dafont.com/kongtext.font)                   |
+| Chinoés  | [Zpix (最像素)](https://github.com/SolidZORO/zpix-pixel-font)      |
```

Comment:
Actually it's "Chinês" here, could you please change it?

Reply:
okay, got it :)